tests/test_utilities.py

```diff
@@ -87,6 +88,37 @@ def test_read(tmpdir, use_pathlib):
     assert multi[1].n_blocks == 2
 
 
+def test_read_force_ext(tmpdir):
+    fnames = (ex.antfile, ex.planefile, ex.hexbeamfile, ex.spherefile,
+              ex.uniformfile, ex.rectfile)
+    types = (pyvista.PolyData, pyvista.PolyData, pyvista.UnstructuredGrid,
+             pyvista.PolyData, pyvista.UniformGrid, pyvista.RectilinearGrid)
+
+    dummy_extension = '.dummy'
+    for fname, type in zip(fnames, types):
+        root, original_ext = os.path.splitext(fname)
+        _, name = os.path.split(root)
+        new_fname = tmpdir / name + '.' + dummy_extension
+        shutil.copy(fname, new_fname)
+        data = fileio.read(new_fname, force_ext=original_ext)
+        assert isinstance(data, type)
+
+
+def test_read_force_ext_wrong_extension(tmpdir):
+    # try to read a .vtu file as .vts
+    # vtkXMLStructuredGridReader throws an error about the validity of the XML file
+    fname = tmpdir / 'airplane.vtu'
+    ex.load_airplane().cast_to_unstructured_grid().save(fname)
+    with pytest.raises(IOError):
+        fileio.read(fname, force_ext='.vts')
+
+    # try to read a .ply file as .vtm
+    # vtkXMLMultiBlockDataReader throws an error about the validity of the XML file
+    fname = ex.planefile
+    with pytest.raises(IOError):
+        fileio.read(fname, force_ext='.vtm')
```

Comment:
This is the test failing... Generally, we want this logic to work so I suspect something is going awry in `pyvista.read()` where `force_ext` isn't actually forcing the VTS reader. Or maybe the VTS reader being used can handle VTU and return a valid object?

Reply:
Thanks for taking a look. The odd thing is that this test passes locally. @MatthewFlamm the observer code I added to standard_reader_routine (see https://github.com/pyvista/pyvista/blob/9ded7740c309227282d00fcd9e13ffc863847253/pyvista/utilities/fileio.py#L149-L150) should catch the VTK error and raise an actual python error - and it does this on my machine, but not any of the test machines...

Reply:
Some information which might help.

If I individually run `test_read_force_ext_wrong_extension`, it passes:
```
pytest test_utilities.py::test_read_force_ext_wrong_extension
```

If I run this whole test file, the test passes
```
pytest test_utilities.py
```

But if I run _all_ the tests in all files (which is what the CI is doing) test fails!
```
pytest .
```